### DISCUSSION TITLE
findMultiple

### DISCUSSION THREAD

**Discussion by aathi271991** [stephanrauh/ngx-extended-pdf-viewer#2418] [Q&A]
I'm currently working on implementing multiple text search functionality within a findMultiple function, but I'm encountering issues with it, particularly with the highlight-all Texts not displaying correctly

**Comment by stephanrauh** [comment]
Which issues? I've removed my own "find multiple" implementation a long time ago because it brought me into merge conflict hell. However, I can tell you that it's possible to highlight every search result. I remember I had to draw diagrams with paper and pen because the source code confused me, but with some patience and tenacity, you can do it.

**Reply by rmdorsey** [reply]
I was able to search for multiple phrases by wrapping all search phrases into a regex string.  https://github.com/stephanrauh/ngx-extended-pdf-viewer/issues/2339

**Reply by stephanrauh** [reply]
Nice solution!
